src/main/java/org/broadinstitute/consent/http/resources/ElectionResource.java

```diff
@@ -115,4 +118,16 @@ public Response isDataSetElectionOpen(@Context UriInfo info) {
         }
     }
 
+    @GET
+    @Produces("application/json")
+    @RolesAllowed({ADMIN, CHAIRPERSON, MEMBER})
+    @Path("/{electionId}/votes")
+    public Response describeVotesOnElection(@Auth AuthUser authUser, @PathParam("electionId") Integer electionId) {
+        try {
+            return Response.ok().entity(voteService.findVotesByElectionId(electionId)).build();
+        } catch (Exception e) {
+            return createExceptionResponse(e);
+        }
```

Comment:
When I test this with a non-existent election id, I get back an empty list, not a 404 like we describe in the api docs. There are a couple of ways to solve this problem. There are two cases here, the case where there are no votes for an election (generally an error case, but one that is theoretically possible), and the case where there is no election for the provided id. This should be handled in the service class, so will comment more on that line.